code/mob/living/life/blood.dm

```diff
@@ -81,7 +81,7 @@
 
 		owner.blood_volume = max(0, owner.blood_volume) //clean up negative blood amounts here. Lazy fix, but easier than cleaning up every place that blood is removed
 		var/current_blood_amt = owner.blood_volume
-		if (owner.reagents?.total_volume > 0)
+		if (owner.reagents?.total_volume > 0 && src.owner.regens_blood)
```

Comment:
I assume this is so you can't buffer your way out of having 0 blood by loading up on cholesterol/procovertin, but they do both have their own built-in negatives to discourage loading up on massive amounts, so I'm not sure that blocking it here is needed

Reply:
It was more aimed at just natural blood regen actually. I can still remove it if you think it'd be better

Reply:
Blood regen is not handled in that code block though. Natural regen is provided by the spleen (for humans)